fe/fe-core/src/main/java/org/apache/doris/analysis/NativeInsertStmt.java

```diff
@@ -1023,13 +1023,16 @@ private DataSink createDataSink() throws AnalysisException {
         }
         if (targetTable instanceof OlapTable) {
             OlapTableSink sink;
+            final boolean enableSingleReplicaLoad =
+                    analyzer.getContext().getSessionVariable().isEnableMemtableOnSinkNode()
+                    ? false : analyzer.getContext().getSessionVariable().isEnableSingleReplicaInsert();
             if (isGroupCommitStreamLoadSql) {
                 sink = new GroupCommitBlockSink((OlapTable) targetTable, olapTuple,
-                        targetPartitionIds, analyzer.getContext().getSessionVariable().isEnableSingleReplicaInsert(),
+                        targetPartitionIds, enableSingleReplicaLoad,
                         ConnectContext.get().getSessionVariable().getGroupCommit(), 0);
             } else {
                 sink = new OlapTableSink((OlapTable) targetTable, olapTuple, targetPartitionIds,
-                        analyzer.getContext().getSessionVariable().isEnableSingleReplicaInsert());
+                        enableSingleReplicaLoad);
```

Comment:
do we need add this feature to nereids?

Reply:
This is a parameter configured by the user, in this case through session variables.
What do you mean "add this feature to nereids"?

Reply:
Currently there is no `single_replica_load` option in nereids.
This PR is for fixing a problem.
If there is a need, we can create a new PR for that feature.